scripts/avatars/emerg_wds.py

```diff
@@ -0,0 +1,35 @@
+from brownie import chain
+
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+
+def main(sim=False):
```

Comment:
i added a snapshot here to verify withdrawals:
```
    trops = GreatApeSafe(r.badger_wallets.treasury_ops_multisig)

    trops.take_snapshot(
        [
            r.balancer.B_20_BTC_80_BADGER,
            r.balancer.B_50_BADGER_50_RETH,
            r.balancer.bpt_40wbtc_40digg_20graviaura,
            r.treasury_tokens.badgerFRAXBP_f_lp,
        ]
    )
```
```
snapshot result for 0x042B32Ac6b453485e357938bdC38e0340d4b9276:
                                   balance_before             balance_after             balance_delta
symbol                                                                                               
20WBTC-80BADGER             27.149082775210741006  1,357.454138760533741006  1,330.305055985323000000
50rETH-50BADGER              5.406962666590655894    270.348133329533855894    264.941170662943200000
40WBTC-40DIGG-20graviAURA    0.178129933072468065      8.906496653623369065      8.728366720550901000
BADGERFRAX-f               119.074636809456823198  5,953.731840472893823198  5,834.657203663437000000
```

Reply:
included the snap and move to the warroom directory in 9730a2bc0110c978b1c089a6fc6b8707d4357026